docs/content.zh/docs/connectors/table/kinesis.md

```diff
@@ -776,7 +776,7 @@ You can enable and configure EFO with the following properties:
     This is the preferred strategy for the majority of applications.
     However, jobs with parallelism greater than 1 will result in tasks competing to register and acquire the stream consumer ARN.
     For jobs with very large parallelism this can result in an increased start-up time.
-    The describe operation has a limit of 20 [transactions per second](https://docs.aws.amazon.com/kinesis/latest/APIReference/API_DescribeStreamConsumer.html),
```

Comment:
I think this is intentional, the API is called `DescribeStreamConsumer`, could be more explicit and change to 

> The `DescribeStreamConsumer` operation

Reply:
Ah, ok, that makes sense, updated accordingly.